bundles/org.openhab.binding.wlanthermo/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,735 @@
+				<description>Network address of the WlanThermo.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="username" type="text">
+				<label>Username</label>
+				<description>Optional, only required for write access. Default: 'admin'</description>
+				<required>false</required>
+				<default>admin</default>
+			</parameter>
+			<parameter name="password" type="text">
+				<context>password</context>
+				<label>Password</label>
+				<description>Optional, only required for write access. Default: 'admin'</description>
+				<required>false</required>
+				<default>admin</default>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" min="1" step="1" unit="s">
+				<label>Polling Interval</label>
+				<description>Seconds between fetching values from the WlanThermo.</description>
+				<required>true</required>
+				<default>10</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="mini">
+		<label>WlanThermo Mini</label>
+		<description>WlanThermo Mini</description>
+
+		<channel-groups>
+			<channel-group id="system" typeId="cg_system_mini" />
+			<channel-group id="channel0" typeId="cg_temperature_mini0" />
+			<channel-group id="channel1" typeId="cg_temperature_mini1" />
+			<channel-group id="channel2" typeId="cg_temperature_mini2" />
+			<channel-group id="channel3" typeId="cg_temperature_mini3" />
+			<channel-group id="channel4" typeId="cg_temperature_mini4" />
+			<channel-group id="channel5" typeId="cg_temperature_mini5" />
+			<channel-group id="channel6" typeId="cg_temperature_mini6" />
+			<channel-group id="channel7" typeId="cg_temperature_mini7" />
+			<channel-group id="channel8" typeId="cg_temperature_mini8" />
+			<channel-group id="channel9" typeId="cg_temperature_mini9" />
+			<channel-group id="pit1" typeId="cg_pitmaster_mini1" />
+			<channel-group id="pit2" typeId="cg_pitmaster_mini2" />
+		</channel-groups>
+
+		<config-description>
+			<parameter name="ipAddress" type="text">
+				<context>network-address</context>
+				<label>Network Address</label>
+				<description>Network address of the WlanThermo.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" min="1" step="1" unit="s">
+				<label>Polling Interval</label>
+				<description>Seconds between fetching values from the WlanThermo.</description>
+				<required>true</required>
+				<default>10</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<!-- System Group Nano-->
+	<channel-group-type id="cg_system_nano">
+		<label>System Channel</label>
+		<description>This group contains all system channels</description>
+		<channels>
+			<channel id="soc" typeId="system.battery-level" />
+			<channel id="charge" typeId="charging" />
+			<channel id="rssi" typeId="rssi" />
+			<channel id="rssi_signalstrength" typeId="system.signal-strength" />
+			<channel id="online" typeId="online" />
```

Comment:
In general the thing status should represent the online status of the device and not a separate channel. Is there a reason to have separate channel other than a channel can be handled easier?

Reply:
It was requested by users of the binding in order to have a conditional view, e.g. in Basic UI